### ISSUE TITLE
Compiler/bundler breaks formatting of foreging module code.

### ISSUE THREAD

**Issue by wclr** [purescript/purescript#4124]
## Description

I have a foreing.js with the code:
```js
exports.fn = (
  x, y
) => {

};
```
and the compiler while bundling turns it into doesn't respect existing formatting of brackets and arrows:
```
(function(exports) {
  exports.fn = (
    x, y)
    => {

  };
})(PS["BadForeign"] = PS["BadForeign"] || {});
```
putting `)` not in the same line as the `=>`.

While this code is totally legal JS this just cased the problem for me while building with ncc: https://github.com/vercel/ncc/issues/721

## To Reproduce

Create a module with a foreign code above and bundle. (`purs bundle ./output/BadForeign/*.js -o bad-bundle.js`)

## Expected behavior

Respect (at least not break in this strange way) the existing formatting of the foreign module code?

## Additional context

-

## PureScript version

0.14.2

**Comment by rhendric** [comment]
Looks like a bug in language-javascript; I think https://github.com/erikd/language-javascript/blob/376b83afdba022da1462f9814e63f2c5a2b15120/src/Language/JavaScript/Pretty/Printer.hs#L106 needs the `")"` and `rb` swapped.

**Comment by JordanMartinez** [comment]
Looks like this is also resolved via #4255
